lib/provider/todo_provider.dart

```diff
@@ -13,27 +13,36 @@ addNewTodo = function to add a task in the list based on the newDate
  else, a new task will be added*/
 
 class TodoProvider with ChangeNotifier {
-  Map<String, Todo> _todoList = {}; ///Map for _todolist
+  ///Map for _todolist
+  Map<String, Todo> _todoList = {};
```

Comment:
make this private

Reply:
I think this is already private because of underscore"_".

Reply:
I see.